mandatory/2-function-creation.js

```diff
@@ -45,7 +76,15 @@ function canRegister(age) {}
   )
 */
 
-function countReverse(number) {}
+function countReverse(number) {
+  let lastNumber = 1;
+     while (number >= lastNumber){
+    console.log(number)
```

Comment:
I think it's better to remove all console.logs from your pull request, because it might be confusing for people who are reviewing your job.

Reply:
Oh, yes! Thanks! just forgot about that)